src/agenda.ts

```diff
@@ -8,25 +8,29 @@ export class Contact {
         this.surname=theSurname;
         this.address=theAddress;
     }
-    
-    // public displayContact(surname){
-    //     this.surname=surname;
-    //     return this.name + ' ' + this.surname + ' ' + this.address;
-    // }
 
 }
 
 
 export class Agenda {
     contacts: Contact[] = [];
     
-    public addContact(newName: string, newSurname: string, newAddress: string){
-        this.contacts.push(new Contact(newName, newSurname, newAddress));
+    public addContact(Contact){
```

Comment:
Okay. I thought that since I'm passing a Contact object I should write Contact (= Object of the class contact). But this is just a regular variable so I should write "contact: string"?